packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -3377,7 +3379,7 @@ export type MessageThreadSelector = (state: ChatClientState, props: ChatBaseSele
 };
 
 // @public
-export interface MessageThreadStrings {
+export interface MessageThreadStrings extends RichTextStrings {
```

Comment:
Just wondering if there was a decision of nesting this vs being flat?
i.e.
```ts
// current:
export interface MessageThreadStrings extends RichTextStrings {
  ...messageThreadString
}
// vs nested:
export interface MessageThreadStrings {
  richTextSendBox: RichTextStrings,
  ...messageThreadString
}
```

Only concern with `extend` is that we may end up with conflicting top-level strings as its hard to spot what is in the `extends` interface when developing, e.g. something like this could happen:
```tsx
interface MessageThreadStrings extends RichTextStrings, AvatarStrings {
  messageTooltipText: string;
}

interface RichTextStrings { sendButtonTooltipText: string }
interface AvatarStrings { tooltipText: string }

<MessageThread
  strings={{
    tooltipText: 'something' // this only applies to avatar which is not obvious
    messageTooltipText: '',
    sendButtonTooltipText: '' // does this apply to the rich text sendbox button or the simple send box button ? What if rich text sendbox later has the ability to have a "sechduled send" that needs a more complex tooltip
  }}
/>
```

in a  nested world it would look like:
```tsx
<MessageThread
  strings={{
    avatar: { tooltipText: 'something' }
    messageTooltipText: '',
    richTextSendBox: { sendButtonTooltipText: '' }
  }}
/>
```